content/doc/developer/plugin-development/pipeline-integration.adoc

```diff
@@ -2,12 +2,579 @@
 layout: developerguide
 title: Writing Pipeline-Compatible Plugins
 references:
-- url: https://github.com/jenkinsci/pipeline-plugin/blob/master/DEVGUIDE.md
-  title: Plugin Developer Guide
 - url: https://github.com/jenkinsci/workflow-step-api-plugin/blob/master/README.md
   title: Writing Pipeline steps
+- url: https://github.com/jenkinsci/workflow-basic-steps-plugin/blob/master/CORE-STEPS.md
+  title: Core Pipeline steps
```

Comment:
Unclear title in this context.

From the PoV of this doc, the only interesting part would I think be `#interacting-with-build-status`, which is already more or less covered here by `#run-listeners-vs-publishers`. So I think this link could just be dropped.